website/source/docs/commands/catalog/services.html.md.erb

```diff
@@ -50,6 +50,10 @@ Usage: `consul catalog services [options]`
 <%= partial "docs/commands/http_api_options_client" %>
 <%= partial "docs/commands/http_api_options_server" %>
 
+#### Enterprise Options
+
+<%= partial "docs/commands/http_api_namespace_options" %>
```

Comment:
Should we mention that this is only available in 1.7? All the other blurbs so far have

Reply:
I will be adding that note into the partial so it shows up in all the CLI docs